lib/Advisor.js

```diff
@@ -27,7 +27,6 @@ class Advisor {
     this.panel = this.createPanel();
     this.createRequestListeners();
     this.createWindowListeners();
-    this.handleSyncChange();
```

Comment:
I created a more generic `checkForSync` function that I now call at this location. It's also used as the event handler for the pref watcher.